overlord/ifacestate/helpers.go

```diff
@@ -103,6 +104,8 @@ func (m *InterfaceManager) addSnaps() error {
 	for _, snapInfo := range snaps {
 		addImplicitSlots(snapInfo)
 		if err := m.repo.AddSnap(snapInfo); err != nil {
+			logger.Noticef("cannot add snap %q to interface repository: %s", snapInfo.Name(), err)
+			fmt.Fprintf(os.Stderr, "[stderr] cannot add snap %q to interface repository: %s\n", snapInfo.Name(), err)
```

Comment:
I did a quick test locally and it seems its fine to leave printing to stderr out. I get the following when running snapd from my local tree:
```
[log   ] cannot read snap info of snap "vlc" at revision 190: cannot find installed snap "vlc" at revision 190
[stderr] cannot read snap info of snap "vlc" at revision 190: cannot find installed snap "vlc" at revision 190
[log   ] cannot add snap "vlc" to interface repository: snap is broken: cannot read snap "vlc": cannot find installed snap "vlc" at revision 190
[stderr] cannot add snap "vlc" to interface repository: snap is broken: cannot read snap "vlc": cannot find installed snap "vlc" at revision 190
[log   ] snap is broken: cannot read snap "vlc": cannot find installed snap "vlc" at revision 190
[log   ] cannot read snap info of snap "vlc" at revision 190: cannot find installed snap "vlc" at revision 190
[stderr] cannot read snap info of snap "vlc" at revision 190: cannot find installed snap "vlc" at revision 190
```
which indicates that what we get on the logger is also what we get on stderr so that seems redundant.

Reply:
+1, I'll rely on logger then